src/app/components/publiclists/teamlist/teamcard/teamcard.component.html

```diff
@@ -1,41 +1,40 @@
 <div class="team-card-container" [class.selected]='isSelected' [class.noexpand] = 'isHost' [class.expand] = '!isHost'>
 <div class="team-card-container-child">
-  <div class="team-content" (click)='selectToggle()'>
-    <i class="fa fa-circle selectstate" *ngIf='!isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-check selectstate" aria-hidden="true" *ngIf='isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-users" aria-hidden="true"></i>
-    <div class="team-details">
-      <span class="title toptitle">
-	    Team:
-      </span>
-      <span *ngIf="teamView['team_url']">
-      	<a class="orange-text" target="_blank" href="{{teamView['team_url']}}">{{teamView['team_name']}}</a>
-      </span>
-      <span *ngIf="!teamView['team_url']">
-	    {{teamView['team_name']}}
-      </span>
-      <br>
-      <span class="title">
-	    Created by:
-      </span>
-      <span>
-	    {{teamView['created_by']}}
-      </span>
+    <div class="team-content" (click)='selectTeamContentToggle()'>
+      <input type="radio" class="with-gap selectPhase" name="teamSelection" (click)='selectTeamToggle()' *ngIf="isHost" id="{{teamView['team_name']}}">
```

Comment:
Hey @Kajol-Kumari You also need to add the condition for if its on challenge page otherwise radio button will not be shown on the participate tab of challenge page.

Reply:
yes @Sanji515 added the required condition.